pcdsdevices/rtds_ebd.py

```diff
@@ -44,6 +46,27 @@ def _do_move(self, state):
             self.out_cmd.put(1)
 
 
+class RTDSX0ThreeStage(BaseInterface, LightpathMixin):
+    """Three stages X,Y,Z, for solid drilling experiments"""
+
+    mmsx = Cpt(BeckhoffAxis, ':MMS:X', kind='normal')
+    mmsy = Cpt(BeckhoffAxis, ':MMS:Y', kind='normal')
+    mmsz = Cpt(BeckhoffAxis, ':MMS:Z', kind='normal')
+
+    # Determine if the y stage is at it's outer limit switch
+    open_limit = Cpt(EpicsSignalRO, ':MMS:Y.HLS', kind='normal',
```

Comment:
Not much wrong with a single duplicated PV connection here, but I'm curious: is there any way with the `LightpathMixin` to access subcomponents? The Y high limit is a PV connection that already existed under `mmsy.high_limit_switch`

Reply:
Interesting question, I guess it didn't occur to me that the high_limit_switch PV may be accessible without separately reading it into Cpt as I did here. @tangkong, does doing it the way Zach mentions here seem feasible/preferable?

Reply:
Yes, you can list `['mmsy.high_limit_switch']` as one of the `lightpath_cpts`.  Some pcdsdevices do this already I think
https://github.com/pcdshub/pcdsdevices/blob/3bd3b5c121c2cabac4238a4da05bfd5f9cb2b2b0/pcdsdevices/ccm.py#L941

I do think that would be preferable, if only to clean up the class definition a bit

Reply:
ah okay, for the same reason as before, making this change now would require waiting for another chance to test, so I think If it seems okay I will leave it as-is for now, and keep this in mind moving forward!

Reply:
I largely wanted to make sure this was at least possible :+1:
One extra PV in a device with dozens isn't a big deal